.circleci/config.yml

```diff
@@ -220,12 +217,12 @@ commands:
           name: Write cache key
           command: |
             echo <<parameters.executor>> > /tmp/cachefile
-            if [ "<<parameters.determined-common>>" = "true" ]; then
-              cat common/setup.py >> /tmp/cachefile
-            fi
             if [ "<<parameters.determined>>" = "true" ]; then
               cat harness/setup.py >> /tmp/cachefile
             fi
+            if [ "<<parameters.determined-common>>" = "true" ]; then
+              cat common/setup.py >> /tmp/cachefile
+            fi
```

Comment:
seems like we can just remove the explicit treatment of `determined_{common,cli,deploy}` throughout our ci.  If our own tests don't pass without the dummy packages, we probably have a problem.

Reply:
We have some code which bundles the wheels for these packages into master package: https://github.com/determined-ai/determined/blob/master/master/Makefile#L83
I assumed the users of this package may want to install from these wheels (instead of pypi), and they may want to have the legacy package for compatibility reasons. Is that the case? if not, then yes we can simplify it.

Reply:
nah, users won't ever interact with that.  We do package it into wheels for the master, and the master basically injects those wheels into the environment before every training, notebook, tensorboard, or shell runs.

Reply:
done.